game/views.py

```
@@ -549,26 +549,59 @@ def logout_view(request):
+                ]
+            })
+
+        return render(request, 'game/stats.html', {
+            'recent': recent,
+            'ai_total': ai_total,
+            'user_ai_wins': user_ai_wins,
+            'ai_wins': ai_wins,
+            'ai_draws': ai_draws,
+            'win_percentage': round(win_percentage, 2),
+        })
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)